bigtable-hbase-2.x-parent/bigtable-hbase-2.x/src/main/java/com/google/cloud/bigtable/hbase2_x/BigtableAsyncAdmin.java

```diff
@@ -464,12 +455,8 @@ private TableName snapshotExists(String snapshotName,
     if (!preserveSplits) {
       LOG.info("truncate will preserveSplits. The passed in variable is ignored.");
     }
-    DropRowRangeRequest request = DropRowRangeRequest
-        .newBuilder()
-        .setDeleteAllDataFromTable(true)
-        .setName(bigtableInstanceName.toTableNameStr(tableName.getNameAsString()))
-        .build();
-    return bigtableTableAdminClient.dropRowRangeAsync(request).thenApply(r -> null);
+
+    return bigtableTableAdminClient.dropRowRangeAsync(tableName.getNameAsString(), "");
```

Comment:
We should add a `bigtableTableAdminClient.truncateTable(Async)` method eventually.  For now, I guess this is ok.

Reply:
I have made a note of it, Once GCJ adapter is done then we can add this.